fieldservice/models/res_partner.py

```diff
@@ -57,10 +57,13 @@ def action_open_owned_locations(self):
 
     def _convert_fsm_location(self):
         wiz = self.env["fsm.wizard"]
+        location_model = self.env["fsm.location"]
+        if not self.env.user.has_group("fieldservice.group_fsm_user_own"):
```

Comment:
No need to do this. Put always `sudo`, as since 13+, no user spoofing is done. Only the permission part.